lib/topological_inventory/orchestrator/health_checker.rb

```diff
@@ -0,0 +1,52 @@
+module TopologicalInventory
+  module Orchestrator
+    class HealthChecker
+      include Logging
+
+      attr_accessor :sources, :topology, :k8s
+
+      def initialize(sources, topology, k8s)
+        @sources = sources
+        @topology = topology
+        @k8s = k8s
+      end
+
+      def run
+        loop do
+          errors = []
+
+          errors << check_url(topology)
+          errors << check_url(sources)
+          errors << check_k8s(k8s)
+
+          if errors.any?
+            FileUtils.rm_f("/tmp/healthy")
+          else
+            FileUtils.touch("/tmp/healthy")
+          end
+
+          sleep ENV['HEALTH_CHECK_INTERVAL']&.to_i || 60
```

Comment:
can we move it either to top of the class or as an input arg? I would like to have envs more visible

Reply:
this will result to 0, not to 60 in case of missing env